public/admin/config.yml

```diff
@@ -218,3 +218,22 @@ collections:
                 fields:
                   - { label: Question, name: question, widget: text }
                   - { label: Answer, name: answer, widget: markdown }
+  - name: data-disclaimers
+    label: Data disclaimers
+    format: json
+    description: Data disclaimers
+    slug: '{{slug}}'
+    editor:
+      preview: true
+    files:
+      - name: Disclaimers
```

Comment:
Maybe it would be better to have 2 files in the collection, one for state-wide disclaimers and one more granular by FIPS code (if we add a state FIPS, it would apply to the state page only, but not to its counties), that way we can make the FIPS code mandatory and have more type safety (no possibility of a disclaimer without FIPS or state code). 

Consider also adding the `min` property set to `1` to ensure that the collection is never empty.

Reply:
good thought-- i split it into 2 files. as for the min property, its possible that the collection could be empty (if there are no disclaimers live at a given time)